internal/project/framework/internals/cli/cli.go

```diff
@@ -1,13 +1,14 @@
-{{ .Copyright }}
+/**#bean*/ /*#bean.replace({{ .Copyright }})**/
 package cli
 
 import (
 	"fmt"
 	"os"
 
-	"{{ .PkgPath }}/framework/internals/chalk"
-	"{{ .PkgPath }}/framework/internals/global"
-	"{{ .PkgPath }}/framework/internals/sentry"
+	/**#bean*/
+	"demo/framework/internals/chalk" /*#bean.replace("{{ .PkgPath }}/framework/internals/chalk")**/
```

Comment:
put these two lines in one line

Reply:
`go fmt` command make it, `go fmt` will run before I commit,but I don't know the fmt rules
bean macro still works in two lines